### ISSUE TITLE
Comments on "Getting the Unity Localization Package to work with HoloLens"

### ISSUE THREAD

**Issue by LocalJoost** [LocalJoost/BlogComments#382]
Original article: https://localjoost.github.io/Getting-the-Unity-Localization-Package-to-work-with-HoloLens/

**Comment by BarboraAtVResearch** [comment]
I am just about to start using Unity's Localization package with MRTK myself, and I assume that getting it to work with MRTK's PressabeButton (namely its label) will present an extra challenge :sweat_smile:

**Comment by LocalJoost** [comment]
Not at all! In all those buttons there's just a TextMeshPro that you can localize using LocalizeStringEvent like any other text!
